src/LazyEntityGraph.EntityFrameworkCore.Tests/LazyEntityGraph.EntityFrameworkCore.Tests.csproj

```diff
@@ -6,9 +6,9 @@
     <PackageReference Include="AutoFixture" Version="[4.2.0,5)" />
     <PackageReference Include="AutoFixture.Xunit2" Version="[4.2.0,5)" />
     <PackageReference Include="FluentAssertions" Version="[5.4.1,6)" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="2.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="2.2.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.2.6" />
```

Comment:
I had to update versions because on the previous ones adding specimens to InMemory EF didn't work.